website/templates/base.html

```diff
@@ -56,9 +56,9 @@
                                 {% if request.user.userprofile.avatar %}
                                     <img src="{{ request.user.userprofile.avatar }}">
                                 {% elif request.user.socialaccount_set.all.0.get_avatar_url %}
-                                     <img src="{{request.user.socialaccount_set.all.0.get_avatar_url}}">
+                                    <img src="{{request.user.socialaccount_set.all.0.get_avatar_url}}">
                                 {% else %}
-                                    <img src="{% gravatar request.user.email 20 %}">
+                                    {% gravatar request.user.email 20 %}
```

Comment:
Sorry for not being explicit. What I meant was, we should change that to `<img src="{% gravatar_url request.user.email 20 %}">` to maintain uniformity as this is the style we are following to spit out gravatar image within the app. Since no "class" or any other attribute is associated in this case, `gravatar request.user.email 20 %` works too.

Nevermind! 
Merging 🚀